src/Microsoft.AspNetCore.Hosting/WebHostBuilder.cs

```diff
@@ -151,26 +152,36 @@ public IWebHost Build()
             }
             _webHostBuilt = true;
 
-            // Warn about deprecated environment variables
-            if (Environment.GetEnvironmentVariable("Hosting:Environment") != null)
-            {
-                Console.WriteLine("The environment variable 'Hosting:Environment' is obsolete and has been replaced with 'ASPNETCORE_ENVIRONMENT'");
-            }
+            var hostingServices = BuildCommonServices(out var hostingStartupErrors);
```

Comment:
May should note that this ordering changed to guarantee _options is set.

Reply:
Nah, I don't think we need to doc each minor change like an update for the order of operations. That's what commit histories are for.